lib/commands/nw-test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var chalk = require('chalk');
+var RSVP  = require('rsvp');
+var path = require('path');
+var denodeify = RSVP.denodeify;
+var fs = require('fs');
+
+var readFile  = denodeify(fs.readFile);
+var writeFile = denodeify(fs.writeFile);
+
+module.exports = {
+  name: 'nw:test',
+  description: 'Runs your test suite in NW.js',
+  availableOptions: [
+    { name: 'environment', type: String, default: 'test', aliases: ['e', { 'dev' : 'development' }, { 'prod' : 'production' }] },
+    { name: 'output-path', type: String, default: 'dist/', aliases: ['o'] },
+  ],
+  run: function(options) {
+    var _this = this;
+    var ui = this.ui;
+    
+    ui.startProgress(chalk.green('Building'), chalk.green('.'));
+
+    var buildTask = new this.tasks.Build({
+      ui: this.ui,
+      analytics: this.analytics,
+      project: this.project,
+      environment: options.environment
+    });
+    var testHtmlPath = path.resolve(_this.project.root, 'dist/tests/index.html');
+    
+    return buildTask.run(options).then(function () {
+
+      
+      return readFile(testHtmlPath, { encoding: 'utf8' });
+    })
+    .then(function (rawHTML) {
+
+      rawHTML = rawHTML.replace('base href=\"/\"', 'base href="../"');
+
+      return writeFile(testHtmlPath, rawHTML);
+
+    })
+    .then(function () {
+      // copy test package.json for nwjs
+      fs.createReadStream(path.resolve(_this.project.root, 'tests/package.json')).pipe(fs.createWriteStream(
+        path.resolve(options.outputPath, 'tests/package.json')));
+
+
+      var testTask = new _this.tasks.Test({
+        project: _this.project
+      });
+
+      return testTask.run({
+
+        configFile: path.resolve(_this.project.root, 'tests/testem-nw.json'),
+
+      });
+
+    })
+    .catch(function(err) {
```

Comment:
I'm not sure this is entirely useful. If you want to display some special message to the user when this fails, you could do something like [this](https://github.com/ember-cli/ember-cli/blob/cddcabf0553d5d95bb40654749bccb3fe3ded89d/lib/tasks/build.js#L46-L50). Otherwise, you just wouldn't have a catch and the promise would get rejected. I believe the error would be displayed regardless.